.circleci/config.yml

```diff
@@ -68,20 +68,67 @@ commands:
 # Define a job to be invoked later in a workflow.
 # See: https://circleci.com/docs/2.0/configuration-reference/#jobs
 jobs:
-  build-and-test:
+  init:
     docker:
       - image: sematicai/sematic-ci:latest
     steps:
       - checkout
       - install-ci-pip-deps
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - .pyenv
+            - project
+
+  build-and-lint:
+    docker:
+      - image: sematicai/sematic-ci:latest
+    steps:
+      - attach_workspace:
+          at: /home/circleci
       - do-static-analysis
+      
+  unit-tests:
+    docker:
+      - image: sematicai/sematic-ci:latest
+    steps:
+      - attach_workspace:
+          at: /home/circleci
       - non-coverage-tests
+
+  integration-test:
```

Comment:
I believe the `installatio`n and `integration` tests should be separate jobs, as they do different things.
`integration` should depend on `installation` (with persistence) as a prerequisite.
Can you add a noop for the `integration` job for now, and fill it in later?

Reply:
Yes, that's the plan, though I planned it in a follow-up PR. I will add a noop job for now.

Reply:
Done.